satellite/marketing/marketingweb/server.go

```diff
@@ -30,6 +37,16 @@ type Server struct {
 	listener net.Listener
 	server   http.Server
 }
+// The three pages contained in addPages are pages all templates require
+// This exists in order to limit handler verbosity
+func addPages(assets []string) ([]string){
+	d := dir + "pages/"
```

Comment:
will root or rootPath be more obvious?

Reply:
I agree that rootPath is more obvious but I wanted to use a single letter variable for two reasons

1-limit verbosity on line 44. 
i.e. 
`{d+"base.html", d+"index.html,d+"banner.html"}` is more succinct then 
`{rootPath+"base.html", rootPath+"index.html",rootPath+"banner.html"}`

2-It's also proper convention according to https://talks.golang.org/2014/names.slide#6

Reply:
How abour `rp`?

Reply:
done